openedx/core/djangoapps/course_date_signals/tests.py

```diff
@@ -61,7 +63,6 @@ def test_dates_for_ungraded_assignments(self):
         with self.store.bulk_operations(self.course.id):
             sequence = ItemFactory(parent=self.course, category="sequential")
             vertical = ItemFactory(parent=sequence, category="vertical")
-            sequence = self.store.get_item(sequence.location)
```

Comment:
Isn't the purpose of the test to see if it saved properly and could be read back?

Reply:
updated